Formula/l/lha.rb

```diff
@@ -1,22 +1,11 @@
 class Lha < Formula
   desc "Utility for creating and opening lzh archives"
-  homepage "https://lha.osdn.jp/"
-  # Canonical: https://osdn.net/dl/lha/lha-1.14i-ac20050924p1.tar.gz
-  url "https://dotsrc.dl.osdn.net/osdn/lha/22231/lha-1.14i-ac20050924p1.tar.gz"
-  version "1.14i-ac20050924p1"
-  sha256 "b5261e9f98538816aa9e64791f23cb83f1632ecda61f02e54b6749e9ca5e9ee4"
+  homepage "https://github.com/jca02266/lha/"
+  url "https://github.com/jca02266/lha/archive/refs/tags/release-20211125.tar.gz"
+  version "1.14i-ac20211125"
```

Comment:
where does the `1.14i-ac` bit come from?

Reply:
Part of the tag message:
> LHa for UNIX version 1.14i-ac20211125 (x86_64-w64-mingw32)

Reply:
@SMillerDev Cool, good to ship then!